frameworks/Crystal/amber/shard.yml

```diff
@@ -0,0 +1,19 @@
+name: amber
+version: 0.1.0
+
+authors:
+  - Amber <contact@ambercr.io>
+
+license: MIT
+
+dependencies:
+  amber:
+    github: Amber-Crystal/amber 
+    version: ~> 0.1.4
+
+  granite_orm:
+    github: Amber-Crystal/granite-orm
+    version: ~> 0.6.2
```

Comment:
@faustinoaq I pushed a new release `0.6.3`.  I recommend removing the version from the shard.yml, perform a `shard update` and then check in the `shard.lock` file so it doesn't get updated.